tests/load/test_update_utils.py

```diff
@@ -1,5 +1,274 @@
+import unittest
+import pytest
+
 from scribe_data import utils
 
 
-def test_get_language_qid():
-    assert utils.get_language_qid("french") == "Q150"
+def test_get_scribe_languages():
+    test_case = unittest.TestCase()
+
+    # test for content, not order
```

Comment:
Minor nit here, @m-charlton: for comments we try to do capitalized and fully punctuated if it's its own line, and for inline comments they're not capitalized and have no period as it's here :) Happy to take care of this though!

Reply:
Go for it.